site/docs/rule.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+menu: docs
+title: Rule
+permalink: /docs/rule.html
+---
+
+{: .suppress-error}
+```json
+{
+  "data": ... ,
+  "mark": "rule",
+  "encoding": ... ,
+  ...
+}
+```
+
+The `rule` mark represents each data point as a line. It can be used in two ways. First, as a line that spans the complete width or height of a view. This is useful for adding annotations. Second, a rule can be used to draw a line between two positions.
+
+
+### Usage similar to `tick`
+
+The rule mark can be used similar to a [`tick`](tick.html) mark. The difference is that a rule mark automatically spans the complete width or height of a single view such that no dimension is required. We can sue this for example to show the average price of different stocks.
```

Comment:
I know you try to describe how it normally behaves, but the header "usage similar to tick" is quite weird and it is not really an intended use.   Can you make it just a part (intro paragraph) of "Rules as annotations".

Reply:
Done